Module/Scheduler/DB/pkg_SchedulerLoad.pkb

```diff
@@ -3689,7 +3693,24 @@ is
 
 begin
   pkg_SchedulerMain.getBatch( btr, batchShortName => batchShortName);
-  deleteBatch( batchId => btr.batch_id);
+  if activatedFlag = 0 and btr.activated_flag = 1 then
+    raise_application_error(
+      pkg_Error.IllegalArgument
+      , '����� ' || batchShortName || ' ����������� � �� ����� ���� ������.'
+    );
+  elsif ( activatedFlag = 0 or activatedFlag = 1) and btr.activated_flag = 0 then
+    deleteBatch(
+      batchId => btr.batch_id
+    );
+  elsif activatedFlag = 1 and btr.activated_flag = 1 then
+    pkg_Scheduler.deactivateBatch(
+      batchId => btr.batch_id
+      , operatorId => pkg_operator.getCurrentUserId()
+    );
+    deleteBatch(
+      batchId => btr.batch_id

```

Comment:
No messages on batch delete

Reply:
no else statement, suggestion: add message on batch delete in deleteBatch(batchId=)